evalml/tests/conftest.py

```diff
@@ -1300,3 +1300,42 @@ def tmpdir(tmp_path):
     dir = py.path.local(tmp_path)
     yield dir
     dir.remove(ignore_errors=True)
+
+
+@pytest.fixture
+def df_with_url_and_email():
+    X = pd.DataFrame(
+        {
+            "categorical": ["a", "b", "b", "a", "c"],
+            "numeric": [1, 2, 3, 4, 5],
+            "email": [
+                "abalone_0@gmail.com",
+                "AbaloneRings@yahoo.com",
+                "abalone_2@abalone.com",
+                "$titanic_data%&@hotmail.com",
```

Comment:
these are the most depressing email addresses I've ever seen.  I'm tempted to try to register the hotmail one